candy-machine-core/program/Cargo.toml

```diff
@@ -18,12 +18,12 @@ cpi = ["no-entrypoint"]
 default = []
 
 [dependencies]
-anchor-lang = "0.25.0"
+anchor-lang = "0.26.0"
 arrayref = "0.3.6"
-mpl-token-metadata = { version = "1.5.0", features = ["no-entrypoint"] }
-solana-program = "1.10.29"
-spl-associated-token-account = { version = "~1.0.5", features = ["no-entrypoint"] }
-spl-token = { version = "~3.3.1", features = ["no-entrypoint"] }
+mpl-token-metadata = { version = "1.7.0", features = ["no-entrypoint"] }
+solana-program = "1.14.13"
```

Comment:
Do we want to have this as `1.14`? It seems that the majority of other projects are using `1.14`.

Reply:
Yeah, good point.